ui/component/commentMenuList/view.jsx

```diff
@@ -26,9 +30,12 @@ type Props = {
   // --- select ---
   claim: ?Claim,
   claimIsMine: boolean,
+  isAuthenticated: boolean,
   activeChannelClaim: ?ChannelClaim,
   playingUri: PlayingUri,
   moderationDelegatorsById: { [string]: { global: boolean, delegators: { name: string, claimId: string } } },
+  authorTitle: string,
+  authorCanonicalUri: string,
```

Comment:
Seems like `authorCanoicalUri` should be `?string` -- unresolved, or confirming?
But the only place it is used is gated by a try-catch, so empty `authorCanoicalUri` wouldn't crash, just not append anything.
But still good fix the definition so the next person will be reminded to handle null case if they use the variable.